scripts/usdc-bridge-deployment/README.md

```diff
@@ -0,0 +1,84 @@
+# Bridged USDC standard implementation for Orbit chains
+
+## Background
+
+Circle’s Bridged USDC Standard is a specification and process for deploying a bridged form of USDC on EVM blockchains with optionality for Circle to seamlessly upgrade to native issuance in the future.
+
+We provide custom USDC gateway implementation (for parent and child chain) that follows Bridged USDC Standard. These contracts can be used by new Orbit chains. This solution will NOT be used in existing Arbitrum chains. On parent chain contract `L1USDCGateway` is used in case child chain uses ETH as native currency, or `L1OrbitUSDCGateway` in case child chain uses custom fee token. On child chain `L2USDCGateway` is used. For the USDC token contracts, Circle's referent implementation is used.
+
+This doc describes how to deploy USDC bridge compatible with both Arbitrum's token bridge and Circle’s Bridged USDC Standard.Also steps for transition to native USDC issuance are provided.
+
+## Assumptions
+
+It is assumed there is already USDC token deployed and used on the parent chain. If not, follow the instructions in the Circle's `stablecoin-evm` repo to deploy one.
```

Comment:
I think only Circle should deploy it on the parent chain, if it is not deployed by circle this standard is not applicable

Reply:
Right https://github.com/OffchainLabs/token-bridge-contracts/pull/93/commits/5065de4e9a73d4958dff8b432751f5c6c925edab